frontend/pages/hosts/details/cards/Policies/_styles.scss

```diff
@@ -13,7 +13,6 @@
         }
         .response__header {
           width: 0;
-          border-right: none;
```

Comment:
What's this?

Reply:
I was tired of FE having to one-off fix border-right for second to last header child of only tables that contained an actions/view all host column (which is really buggy when columns disappear but still are considered second to last header). So I switched them all to border-left and put border left none on actions/view all host column.

Reply:
Got it. Works for me.